### PR TITLE
Add zign hash using sha256

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

**Comment by radare** [comment]
tests should be fixed in https://github.com/radare/radare2-regressions/pull/1502

File: libr/include/r_sign.h [hunk 1]
```diff
@@ -43,12 +51,14 @@ typedef struct r_sign_item_t {
 	RSignGraph *graph;
 	ut64 offset;
 	RList *refs;
+	RSignHash *hash;
```
Comment (radare) [inline libr/include/r_sign.h]: why not just have char *hash here? keeping a struct for just one field is unnecessary

**Comment by Maijin** [comment]
ping @cmatthewbrooks

**Comment by cmatthewbrooks** [comment]
Thanks - I'm gonna try to repackage my code soon and will test this out then. The goal will be a separate repo with code and sigs with easy output for other disassemblers.

**Pull request merged**